AutoRest/TestServer/swagger/httpInfrastructure.json

```diff
@@ -2795,6 +2795,9 @@
       }
     },
     "A": {
+      "x-ms-exception": {
```

Comment:
:+1: If `x-ms-client-name` could be applied to model properties too, that would be even better.